src/devicemanager/devicemanager.cpp

```diff
@@ -30,9 +30,8 @@ void DeviceManager::append(const LinkConfiguration& linkConf, const QString& dev
 {
     for(int i{0}; i < _sensors[Connection].size(); i++) {
```

Comment:
`int i{0}` :eyes:

Reply:
This is not related to this patch, we can fix that later.